### PR TITLE
safety_limiter: add delay compensation

### PR DESCRIPTION
**Pull request by at-wat**

### CODE REVIEW

**Comment by at-wat** [comment]
## [[#888](https://travis-ci.org/at-wat/neonavigation/builds/541930367)] FAILED on kinetic

```catkin_make tests``` failed

**Comment by at-wat** [comment]
## [[#889](https://travis-ci.org/at-wat/neonavigation/builds/541940186)] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 6 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 3 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 84 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [[#890](https://travis-ci.org/at-wat/neonavigation/builds/542073986)] PASSED on lunar

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 6 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 3 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 84 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [[#890](https://travis-ci.org/at-wat/neonavigation/builds/542073986)] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 6 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 3 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 84 tests, 0 errors, 0 failures, 0 skipped
```
</details>

File: safety_limiter/src/safety_limiter.cpp [hunk 1]
```diff
@@ -335,10 +338,10 @@ class SafetyLimiterNode
     {
       if (allow_empty_cloud_)
       {
-        return 60.0;
+        return 1.0;
       }
       ROS_WARN_THROTTLE(1.0, "safety_limiter: Empty pointcloud passed.");
-      return DBL_MAX;
```
Comment (at-wat) [inline safety_limiter/src/safety_limiter.cpp]: this was a bug on `allow_empty_cloud:=true`

**Comment by at-wat** [comment]
@ykoga-kyutech please take a look

**Comment by at-wat** [comment]
## [[#891](https://travis-ci.org/at-wat/neonavigation/builds/542081336)] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 6 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 3 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 1 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 86 tests, 0 errors, 0 failures, 0 skipped
```
</details>

File: safety_limiter/src/safety_limiter.cpp [hunk 2]
```diff
@@ -439,24 +442,40 @@ class SafetyLimiterNode
     if (!has_collision)
       return 1.0;
 
-    d_col = std::max<float>(std::abs(d_col) - d_margin_, 0.0);
-    yaw_col = std::max<float>(std::abs(yaw_col) - yaw_margin_, 0.0);
+    // delay compensation:
+    //   d_compensated = d - delay * sqrt(2 * acc * |d_compensated|)
+    //   d_compensated = d + acc * delay^2 - sqrt((acc * delay^2)^2 + 2 * d * acc * delay^2)
```
Comment (ykoga-kyutech) [inline safety_limiter/src/safety_limiter.cpp]: We can obtain 4 results by solving the first equation by Wolfram alpha.
Your solution is the third one in the below solutions. We can ignore other solutions?

solve x x=p_1 - p_2 - sqrt(2-p_3-abs(x)) - Wolfram|Alpha - Google Chrome_355

Comment (at-wat) [reply safety_limiter/src/safety_limiter.cpp]: In the actual calculation, sign of x is handled outside of this formula. Two of these four results are caused by entering both x and abs(x). These cases are not used.
Another one case should mean that the sign of both side of the first formula is different. This is also not a real condition. Negative value of distance is treated outside of this formula.

Comment (ykoga-kyutech) [reply safety_limiter/src/safety_limiter.cpp]: I see, thanks for the detailed explanation. I found that additional restriction x >= 0 results in your solution seems correct  [here](https://www.wolframalpha.com/input/?i=solve+x%3E%3D0,+x%3Da+-+b+*+sqrt(2*c*x)). Will approve.

**Comment by at-wat** [comment]
## [[#894](https://travis-ci.org/at-wat/neonavigation/builds/542687235)] PASSED on melodic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 6 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 3 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 1 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 86 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [[#895](https://travis-ci.org/at-wat/neonavigation/builds/542687277)] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/costmap_cspace/gtest-test_costmap_3d.xml: 6 tests
build/test_results/costmap_cspace/roslint-costmap_cspace.xml: 1 tests
build/test_results/joystick_interrupt/roslint-joystick_interrupt.xml: 1 tests
build/test_results/joystick_interrupt/rostest-test_test_joystick_interrupt_rostest.xml: 1 tests
build/test_results/joystick_interrupt/rosunit-test_joystick_interrupt.xml: 3 tests
build/test_results/map_organizer/roslint-map_organizer.xml: 1 tests
build/test_results/neonavigation_common/roslint-neonavigation_common.xml: 1 tests
build/test_results/obj_to_pointcloud/roslint-obj_to_pointcloud.xml: 1 tests
build/test_results/planner_cspace/gtest-test_blockmem_gridmap.xml: 3 tests
build/test_results/planner_cspace/gtest-test_cyclic_vec.xml: 7 tests
build/test_results/planner_cspace/gtest-test_grid_metric_converter.xml: 2 tests
build/test_results/planner_cspace/gtest-test_motion_cache.xml: 1 tests
build/test_results/planner_cspace/roslint-planner_cspace.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_abort_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_costmap_watchdog_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_compat_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml: 1 tests
build/test_results/planner_cspace/rostest-test_test_preempt_rostest.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_abort.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_costmap_watchdog.xml: 1 tests
build/test_results/planner_cspace/rosunit-test_navigate.xml: 2 tests
build/test_results/planner_cspace/rosunit-test_preempt.xml: 1 tests
build/test_results/safety_limiter/roslint-safety_limiter.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter2_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_compat_rostest.xml: 1 tests
build/test_results/safety_limiter/rostest-test_test_safety_limiter_rostest.xml: 1 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter.xml: 9 tests
build/test_results/safety_limiter/rosunit-test_safety_limiter2.xml: 1 tests
build/test_results/track_odometry/gtest-test_tf_projection.xml: 1 tests
build/test_results/track_odometry/roslint-track_odometry.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_tf_projection_rostest.xml: 1 tests
build/test_results/track_odometry/rostest-test_test_track_odometry_rostest.xml: 1 tests
build/test_results/track_odometry/rosunit-test_tf_projection_node.xml: 3 tests
build/test_results/track_odometry/rosunit-test_track_odometry.xml: 5 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_filter.xml: 3 tests
build/test_results/trajectory_tracker/gtest-test_trajectory_tracker_path2d.xml: 4 tests
build/test_results/trajectory_tracker/roslint-trajectory_tracker.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_recorder_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rostest-test_test_trajectory_tracker_rostest.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_recorder.xml: 1 tests
build/test_results/trajectory_tracker/rosunit-test_trajectory_tracker.xml: 7 tests
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 86 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Pull request merged**